tests/bats/tests.bats

```diff
@@ -74,8 +77,9 @@ function create_tree() {
 @test "blk spt" {
   create_tree
   rm -f test.iso
-  genisoimage -U -J -f -joliet-long -r -allow-lowercase -allow-multidot -o test.iso test_dir
+  run genisoimage -U -J -f -joliet-long -r -allow-lowercase -allow-multidot -o test.iso test_dir
```

Comment:
Actually without `run` is fine since status or output is not being used, unless you want to ignore if `genisoimage` fails.

Reply:
> Actually without `run` is fine since status or output is not being used, unless you want to ignore if `genisoimage` fails.

Yes, would like to ignore it. The issue is that the output of genisoimage is huge, and without the run it's jsut printed without the `run`, making things very confusing.